app/person/middleware/index.js

```diff
@@ -1,4 +1,5 @@
 const setBreadcrumb = require('./set-breadcrumb')
+const setIdentityBar = require('./set-identity-bar')
```

Comment:
It's not really a _defined_ pattern but I was trying to find a way to distinguish middleware that sets locals over a property on `req` and I started [prefixing the filenames with `locals`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/main/app/move/app/view/middleware/locals.identity-bar.js).

Reply:
Good point, I've amended the PR to use this pattern.